src/qt/networkstyle.cpp

```diff
@@ -85,8 +88,8 @@ const NetworkStyle *NetworkStyle::instantiate(const QString &networkId)
                     network_styles[x].appName,
                     network_styles[x].iconColorHueShift,
                     network_styles[x].iconColorSaturationReduction,
-                    network_styles[x].titleAddText);
+                    titleAddText.c_str());
         }
     }
-    return nullptr;
+    return new NetworkStyle(strprintf("%s-%s", QAPP_APP_NAME_CUSTOM, networkId).c_str(), 250, 30, titleAddText.c_str());
```

Comment:
in commit dba00d2b04c3bf0fd4d31cca662d262c459f2ebb:

This is dead code and I'd prefer if the "default purple" and "custom chain" concept were left for the testchains pull request. This is not required for signet (apart from adding dead code).

I am fine with the `titleAddText` change in this commit, though. So you can either remove the commit or strip it of the concept of a custom chain.

Reply:
I guess my idea was for signet to reuse this color, perhaps change the tittle from custom to signet if it comes first, and if not just reuse the custom name for all custom chains signet or not.

Reply:
Wouldn't that also require 2f5796504a5d984276931da974436a8a5d352f74?

Reply:
hopefully fixed